app/uploaders/evss_claim_document_uploader.rb

```diff
@@ -13,7 +13,7 @@ def initialize(user_uuid, tracked_item_id)
   end
 
   def store_dir
-    store_dir = "disability_claim_documents/#{@user_uuid}"
+    store_dir = "evss_claim_documents/#{@user_uuid}"
```

Comment:
I think the solution for queued tasks is to have all new jobs use new classes but leave the old classes with old upload location until all the old jobs are processed and remove them in a separate production push.